Libs/MRML/Widgets/qMRMLLayoutManager.cxx

```diff
@@ -1330,3 +1330,81 @@ QWidget* qMRMLLayoutManager::viewWidget(vtkMRMLNode* viewNode) const
 
   return d->viewWidget(viewNode);
 }
+
+//------------------------------------------------------------------------------
+void qMRMLLayoutManager::pauseRender(bool pause)
+{
+  // Note: views that are instantiated between pauseRender() and resumeRender() calls will
+  // not be affected by the specified pause state
+
+  Q_D(qMRMLLayoutManager);
+
+  qMRMLLayoutManagerPrivate::PausedViewVector currentPausedViews;
+
+  QString sliceNodeType = "vtkMRMLSliceNode";
+  qMRMLLayoutViewFactory* sliceViewFactory = this->mrmlViewFactory(sliceNodeType);
+  foreach(const QString& viewName, sliceViewFactory->viewNodeNames())
+    {
+    ctkVTKAbstractView* view = (ctkVTKAbstractView*)this->sliceWidget(viewName)->sliceView();
+    qMRMLLayoutManagerPrivate::PausedView pausedView;
+    pausedView.Name = viewName.toStdString();
```

Comment:
try to minimize conversions between QString and std::string. In Qt classes, Qt types are preferred, but if you only ever need to store std type in a variable then it is OK to use std.